### DISCUSSION TITLE
Scanning attributes of dynamic component

### DISCUSSION THREAD

**Discussion by karlvignon** [finsweet/wized-roadmap#70] [RFCs]
I just encountered an issue that might be worth reporting to the developers. I had trouble scanning the attributes of my logout button. I believe the problem was that the button with the Wized attribute was inside a component that was displayed dynamically, based on whether the user was logged into my website. 

My guess is that Wized's attribute scanning process only looks for attributes on the initial loading page. When the loading page ended and the home page was displayed, Wized was unable to find the attributes and informed me that none were present. 

To resolve this, I had to create a separate Webflow page without authentication and place the component there. In Wized, I navigated to this page, and I was finally able to detect the attribute.

**Comment by alexiglesias93** [comment]
>that the button with the Wized attribute was inside a component that was displayed dynamically, based on whether the user was logged into my website.

How is the dynamic display of that component managed? Is it managed by Wized? Or by Webflow or another party?
Can you give use a bit more context on that component?

Thank you!

CC @besart-finsweet

**Comment by besart-finsweet** [comment]
In this case, can we implement a way to let users to add elements from Wized?

**Reply by alexiglesias93** [reply]
I don't think we should do this, defining elements should be done 100% in Webflow otherwise we risk having multiple sources of truth 🤔

**Reply by besart-finsweet** [reply]
I think this is not a big problem, it is just an attribute name, we can have a field that tell which are already in webflow which are only on wized.
